build/x86_64_debian/Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN /bin/bash -l -c "apt-get update -y && apt-get install -y \
         curl \
   && gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 \
   && curl -sSL https://get.rvm.io | /bin/bash -s stable \
```

Comment:
Let's move this to just before the "rvm" command. Also see my other comment about the scope of the pipe.

Reply:
Done.